ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-fcos.yaml

```diff
@@ -3,6 +3,14 @@ base_images:
     name: "4.3"
     namespace: origin
     tag: base
+  base-machine:
+    name: fedora
+    namespace: openshift
+    tag: "29"
```

Comment:
am i missing s'thing @LorbusChris w.r.t the version ? i thought will be 31 ?

Reply:
29 has proven to work so far, we'll update this soon!

Reply:
I think we can actually switch to coreos-assembler now, but haven't yet got to it.

Reply:
I'm planning to port these changes to 4.2+ OCP branches. Should I wait for the switch to `cosa` with that?